tests/integration/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.1'
+
+services:
+  haproxy:
+    image: haproxy:2.3
```

Comment:
have you check there is no braking changes from v2.2 ? according to[ our docs](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/haproxy-monitoring-integration/) the las supported verison is v2.2. If there is not it would be nice to bump the version on the docs.

Reply:
I changed it to v2..2